### ISSUE TITLE
[APP SUBMITTED]: Error while searching TorrentDay, skipping: No JSON object could be decoded

### ISSUE THREAD

**Issue by Rouzax** [SiCKRAGETV/sickrage-issues#1850]
### INFO
Python Version: **2.7.9 (default, Dec 10 2014, 12:24:55) [MSC v.1500 32 bit (Intel)]**
Operating System: **Windows-2012Server-6.2.9200**
Branch: **master**
Commit: SiCKRAGETV/SickRage@52a6c31cdf545c4e2fd22925a92a3d0824c429cf
Link to Log: https://gist.github.com/bd5fc28bf21fd963e78c
### ERROR
```
SEARCHQUEUE-BACKLOG-277964 :: [TorrentDay] :: Unable to obtain cookie for TorrentDay
SEARCHQUEUE-BACKLOG-121361 :: [TorrentDay] :: Error while searching TorrentDay, skipping: No JSON object could be decoded
SEARCHQUEUE-BACKLOG-277964 :: Error while searching TorrentDay, skipping: No JSON object could be decoded
```
---
_STAFF NOTIFIED_: @SiCKRAGETV/owners @SiCKRAGETV/moderators

**Comment by neoatomic** [comment]
@Rouzax Please continue the discussion at https://github.com/SiCKRAGETV/sickrage-issues/issues/1823 so we have all the info/logs in one place.

**Comment by miigotu** [comment]
Committed a fix in SiCKRAGETV/SickRage#1978 for Torrentday not working even without proxy, for now proxy still probably won't work.
